docs/firmware-update.md

```diff
@@ -2,6 +2,8 @@
 
 The Firmware Update menu allows you to update the programming on a Ping device.
 
+## Automatic Firmware Update
+
 Steps to flash the device:
```

Comment:
It seems like this should be outside (before) both subtopics ("Automatic" and "Manual Firmware Update").

Reply:
doin